app/helpers/application_helper.rb

```diff
@@ -30,4 +30,23 @@ def file_size_for(file, decimal_places=2)
   def format_url_safe(text)
     ERB::Util.url_encode(text)
   end
-end
+
+  def any_fields_present(type, total)
+    if type == 'file'
```

Comment:
Or better yet, move the thing you're testing as true/false outside the block altogether...

```
def any_fields_present(type, total=5)
  file_fields_present = lambda { |n| cms_block_content("file_#{n + 1}".to_sym).try(:file)&.present? }
  url_fields_present = lambda { |n| cms_block_content("url_#{n + 1}".to_sym)&.present? }
  any_data_for_field = type == 'file' ? file_fields_present : url_fields_present
  
  total.times do |n|
    return_true if any_data_for_field.call(n)
  end

  return false
end
```